data-prepper-plugins/aws-sqs-common/src/main/java/org/opensearch/dataprepper/plugins/aws/sqs/common/metrics/SqsMetrics.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.aws.sqs.common.metrics;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.Timer;
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+
+public class SqsMetrics {
+
+    public static final String SQS_MESSAGES_RECEIVED_METRIC_NAME = "sqsMessagesReceived";
+
+    public static final String SQS_MESSAGES_DELETED_METRIC_NAME = "sqsMessagesDeleted";
+
+    public static final String SQS_MESSAGES_FAILED_METRIC_NAME = "sqsMessagesFailed";
```

Comment:
I think we should rename it to `sqsReceiveMessageFailed` as this is tracking number of failed receive requests. Currently this looks like we are tracking number of SQS messages we filed to process.

Reply:
Done.